src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,108 @@
+package parser;
+
+import com.google.gson.JsonElement;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Test
+public class JsonParserTest {
+
+    private static final double BOOK_PRICE = 25.2;
+    private static final double SOFT_PRICE = 99.0;
+    private static final String PATH_TO_CART_LIST = "/Users/dmitrymandrik/Documents/training/testng-maven/src/test/java/datasets/carts.txt";
+    private static final String PATH_TO_CART = "/Users/dmitrymandrik/Documents/training/testng-maven/src/main/resources/john-cart.json";
```

Comment:
Change absolute paths to relative, this code will work only on your PC

Reply:
oops)